README.md

```diff
@@ -2,6 +2,12 @@
 
 :rocket: Welcome to the home of the Hoodie Editorial Team! :rocket:
 
+## Mission Statement
+
+The Hoodie Editorial Team exists to support [Hoodie’s Initiatives](http://hood.ie/initiatives/) ([Offline First](http://hood.ie/initiatives/#offline-first), [noBackend](http://hood.ie/initiatives/#nobackend), [Hoodie is Dreamcode](http://hood.ie/initiatives/#dreamcode), [tools](http://hood.ie/initiatives/#tools) and products, and encouraging [Non-Coding Experts in Open Source](http://hood.ie/initiatives/#noncoders)) through writing documentation, writing content for a website or blog to design, editing, public relations, social media and much more.
```

Comment:
Love it <3

As I read this, I wonder if we can change "Non-Coding Experts in Open Source" to "Non-Coding Contributors in Open Source", also on http://hood.ie/initiatives/#noncoders? Any thoughts?